cogniac/app.py

```diff
@@ -230,7 +230,7 @@ def post_feedback(self, media_id, subjects):
         subjects (list of dicts):      Subject-media association dictionaries of the form:
 
             subject_uid:               Subject UID
-            result (str):              Either 'True', 'False', 'Uncertain'
+            result (str):              One of 'True', 'False', 'Uncertain', 'Sidelined'
```

Comment:
do we sill support "uncertain" in the backend?  should probably remove references to here here

Reply:
My impression was that support for it was incomplete and it's not actually being used. I saw scattered references to it, but haven't come across business logic that consumes it for anything in particular. Seems like it may have been in the works as something like what Sidelining will do.

I can take a look. It would be nice to disallow that value and remove reference to it clarify the feedback pathway if it really isn't being used.